taichi/backends/opengl/opengl_kernel_launcher.h

```diff
@@ -10,16 +10,18 @@ TLANG_NAMESPACE_BEGIN
 namespace opengl {
 
 struct CompiledProgram;
-struct GLSLLauncherImpl;
-struct GLSLLauncher;
+struct OpenGLRuntimeImpl;
+struct OpenGLRuntime;
 class GLBuffer;
 
-struct GLSLLauncher {
-  std::unique_ptr<GLSLLauncherImpl> impl;
+struct OpenGLRuntime {
```

Comment:
super nit: `OpenGlRuntime` (let's stick to the convention of capitalizing only the first character)